tasks/managed/push-snapshot/push-snapshot.yaml

```diff
@@ -325,32 +325,40 @@ spec:
               echo "Error: Source container ${sourceContainer} not found!"
               exit 1
             fi
-            # Push the source image with the source tag here. The source image will be
```

Comment:
I think this comment is helpful and should be in the new version of the task too

Reply:
Done.